.github/workflows/release.yaml

```diff
@@ -0,0 +1,243 @@
+        with:
+          name: conformance-test-files
+          path: conformance-test-files.tar
+
+      - name: Bundle the build report
+        if: failure()
+        run: find . -type d -name 'reports' | zip -@ -r build-reports.zip
+
+      - name: Upload the build report
+        if: failure()
+        uses: actions/upload-artifact@v2
+        with:
+          name: error-report
+          path: build-reports.zip
+
+  conformance:
+    needs: build
+    strategy:
+      matrix:
+        platform: [jvm, js, linux]
+      fail-fast: false
+    runs-on: ubuntu-18.04
+
+    steps:
+      - name: Install Java ${{ env.JAVA_VERSION }}
+        if: matrix.platform == 'jvm'
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ env.JAVA_VERSION }}
+
+      - name: Install Node.js ${{ env.NODEJS_VERSION }}
+        if: matrix.platform == 'js'
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ env.NODEJS_VERSION }}
+
+      - name: Cache protobuf ${{ env.PROTOBUF_VERSION }}
+        uses: actions/cache@v2
+        with:
+          path: ~/protobuf
+          key: ${{ runner.os }}-protobuf-${{ env.PROTOBUF_VERSION }}
+
+      - name: Download conformance test files
+        uses: actions/download-artifact@v2
+        with:
+          name: conformance-test-files
+
+      - name: Extract conformance test files
+        run: tar -xvf conformance-test-files.tar
+
+      - name: Run ${{ matrix.platform }} conformance tests
+        run: |
+          export CONF_TEST_PATH=~/protobuf/conformance/conformance-test-runner
+          ./conformance/test-conformance.sh ${{ matrix.platform }}
+
+  build-examples:
+    needs: build
+    runs-on: ubuntu-18.04
+
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@v2
+
+      - name: Validate Gradle Wrapper
+        uses: gradle/wrapper-validation-action@v1
+
+      - name: Download maven artifacts
+        uses: actions/download-artifact@v2
+        with:
+          name: maven-artifacts
+          path: ~/.m2/repository
+
+      - name: Cache gradle
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-examples-${{ hashFiles('examples/**/*.gradle*') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-examples-
+
+      - name: Install Java ${{ env.JAVA_VERSION }}
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ env.JAVA_VERSION }}
+
+      - name: Build all examples
+        run: |
+          for d in examples/*; do
+            set -e
+            [ -d "$d" ] || continue
+            echo "::group::Build $d"
+            pushd "$d" >/dev/null
+            ../../gradlew build
+            popd >/dev/null
+            echo "::endgroup::"
+          done
+
+  publish:
+    needs: [build, conformance, build-examples]
+    runs-on: ubuntu-18.04
+
+    steps:
```

Comment:
Migrated the steps from `.cicleci/config.yml` and modified them by looking at the existing steps in this file.
https://github.com/streem/pbandk/blob/c4a15824ba28797ed45ff517fbd386d928dcb297/.circleci/config.yml#L249

Reply:
Please also delete the `.circleci/config.yml` file as part of this PR so that the next release doesn't trigger both CircleCI and GitHub Actions jobs.